itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosIncrementalLoadAcidTables.java

```diff
@@ -243,6 +248,53 @@ public void testReplCM() throws Throwable {
             Lists.newArrayList(result, result));
   }
 
+  @Test
+  public void testReplCommitTransactionOnSourceDelete() throws Throwable {
+    String tableName = "testReplCommitTransactionOnSourceDelete";
+    String[] result = new String[] { "5" };
+
+    // Do a bootstrap dump.
+    WarehouseInstance.Tuple bootStrapDump = primary.dump(primaryDbName);
+    replica.load(replicatedDbName, primaryDbName).run("REPL STATUS " + replicatedDbName)
+        .verifyResult(bootStrapDump.lastReplicationId);
+
+    // Add some data to the table & do a incremental dump.
+    ReplicationTestUtils.insertRecords(primary, primaryDbName, primaryDbNameExtra, tableName, null, false,
+        ReplicationTestUtils.OperationType.REPL_TEST_ACID_INSERT);
+    WarehouseInstance.Tuple incrementalDump = primary.dump(primaryDbName);
```

Comment:
Can you please add the tables with following property:
- ORC Format (I think covered)
- bucketed 
- text input folder
All these tables should have a drop table use case like you are targeting now?